taxi/views.py

```diff
@@ -1,3 +1,22 @@
+from django.http import HttpRequest, HttpResponse
+
+from taxi.models import Car, Manufacturer, Driver
+
 from django.shortcuts import render
 
-# Create your views here.
+
+MENU = ["Home page", "Manufacturers", "Cars", "Drivers"]
+EMPTY_MENU = {item: "" for item in MENU}
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_cars = Car.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    context = {
+        "num_drivers": num_drivers,
+        "num_cars": num_cars,
+        "num_manufacturers": num_manufacturers,
+        "menu": EMPTY_MENU,
```

Comment:
Why not just do?
`{item: "#" for item in ["Home page", "Manufacturers", "Cars", "Drivers"]}`